### DISCUSSION TITLE
Sliders setting on dashboard, value not visible

### DISCUSSION THREAD

**Discussion by EndreOlah** [home-assistant/frontend#15700] [Dashboard] [answered]
Hi,

on the dashboard when I want to set a blue with a slider, the new value in the bubble is not visible. It is hard to set the value correctly right the first time.

Could you solve this problem, please?

Thanks

       Endre

<img width="1208" alt="Screenshot 2023-03-04 at 12 00 28" src="https://user-images.githubusercontent.com/6631173/222896336-9e28adb8-badb-4d02-8688-747837ef2642.png">

**Comment by ildar170975** [comment, marked as answer]
https://github.com/home-assistant/frontend/issues/15684
